cincinnati/src/plugins/internal/graph_builder/commons.rs

```diff
@@ -0,0 +1,27 @@
+#[cfg(test)]
+pub mod tests {
+    //! Common functionality for graph-builder tests
+
+    use crate as cincinnati;
+
+    use cincinnati::plugins::internal::graph_builder::release;
+    use cincinnati::plugins::internal::graph_builder::release_scrape_dockerv2::registry;
+
+    fn init_logger() {
```

Comment:
RLS warns that this is a dead code - needs `#![allow(dead_code)]`?

Reply:
Note that this is gated by `[cfg(test)]`. It probably depends on the configuration, as I don't get a warning from the RLS run by vscode. I also don't see any warnings when building any of the crates.

Reply:
Odd, must be must config settings